sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,5 @@
+select "ctry"."name" as "country",
+       count(*) as "numberOfCities"
+  from "countries" as "ctry"
+  join "cities" as "c" using ("countryId")
+group by "ctry"."name";
```

Comment:
It would be best to group by countryId to cover the unlikely case where two countries have the same name.